tests/Microsoft.Identity.Test.Unit/PublicApiTests/ConfidentialClientApplicationTests.cs

```diff
@@ -1561,6 +1561,52 @@ public async Task AcquireTokenForClientAuthorityCheckTestAsync()
                 Assert.IsTrue(log.Contains(MsalErrorMessage.ClientCredentialWrongAuthority));
             }
         }
+
+        [TestMethod]
```

Comment:
nit: the 2 tests are identical except that one passes in "" and one passes in null as acc_id. To avoid code duplication, you can use a parametrized test, smth like: 

```csharp
[TestMethod]
[DataRow("")]
[DataRow(null)]
public async Task ValidateGetAccountAsyncWithNullAccountIdAsync(string accountId)
{
 // test logic
}
```

Of course, you can also just do: 

```csharp
var acc = await app.GetAccountAsync("").ConfigureAwait(false);
// assert
var acc2 = await app.GetAccountAsync(null).ConfigureAwait(false);
```

Reply:
Thanks @bgavrilMS, updated tests per suggestion.